short/index.html

```diff
@@ -357,27 +374,16 @@
 
 *   Write **lots** of tests for scientific core of the code
 *   Start with many fewer tests for other project components
-    *   command-line interface
-    *   file I/O
-    *   other "boring" and "unimportant" code
 *   When bugs are encountered:
+    *   write new regression tests specific to that bug  <!-- #tdd #ftw -->
     *   fix the observed bug
-    *   write new regression tests specific to that bug
     *   get on with more important things
 </section>
 
 <section data-markdown>
 **[Stupidity-Driven Development](http://ivory.idyll.org/blog/2014-research-coding.html)**
 
-*   Avoids wasting time writing tests for bugs that:
-    *   never appear, or appear rarely
-    *   don't affect correctness of results
-*   Order of priorities
-    *   correctness
-    *   performance
-    *   user experience
-    *   ...
-    *   beautiful code
+Avoids wasting time writing tests for bugs that never appear!
```

Comment:
I agree they're important points, but 90-120 minutes limits the number of important points you can make, and the amount of text the audience can handle. In the spirit of the material, I had to make some compromises. :-)